### ISSUE TITLE
Outdated installation

### ISSUE THREAD

**Issue by ghost** [grid-unsw/ifc2indoorgml#1]
It seems like the installation instructions are a little outdated and rather hard to follow. Any updated planned in that regard? I cant reproduce the installation.

**Comment by abdoulayediak** [comment]
Can you be more specific please? Tell us more about your system (OS, C++ environment. etc.) and issues you are facing (which part of the setup aren't you able to replicate).

**Comment by ghost** [comment]
Im using Windows 64 bit.

Finding a Qt5 <= 5.10 Version was difficult at first but after i found it the installation menu was completely different and i could not select components like qscript.

For Step 5 i could not find enough instrcutions to complete the steps myself.

For Step 3 i did not fully understand what to do after downloading it.
